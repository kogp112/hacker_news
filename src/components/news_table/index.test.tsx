import * as React from "react";
import * as renderer from "react-test-renderer";
import NewsTable from "./index";

let container = null;

beforeEach(() => {
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  container = null;
});

it("should render news table", () => {
  const newsIds = ["1","2","3"]

  const component = renderer
  .create(<NewsTable newsIds={newsIds} />)
  .toJSON();
  expect(component).toMatchSnapshot();
});