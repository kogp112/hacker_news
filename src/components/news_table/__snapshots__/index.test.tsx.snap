// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render news table 1`] = `
<div>
  <div
    className="cardgroup"
  >
    <div
      className="card_0"
    >
      <div
        onClick={[Function]}
      >
        <p
          className="container"
        />
      </div>
    </div>
    <div
      className="card_1"
    >
      <div
        onClick={[Function]}
      >
        <p
          className="container"
        />
      </div>
    </div>
    <div
      className="card_2"
    >
      <div
        onClick={[Function]}
      >
        <p
          className="container"
        />
      </div>
    </div>
  </div>
  <div>
    <p
      className="subtitle"
    >
      3 or less
    </p>
  </div>
  <div
    className="listarea"
  >
    <ul
      className="ul"
    />
  </div>
</div>
`;
